jenkins/check-for-build.jenkinsfile

```diff
@@ -15,8 +15,8 @@ pipeline {
             H/10 * * * * %INPUT_MANIFEST=2.0.2/opensearch-2.0.2.yml;TARGET_JOB_NAME=distribution-build-opensearch
```

Comment:
Can we also reduce the frequency for this since it creates a lot of chaos now.

Reply:
+1 also 1.3.4

Reply:
Thanks.